src/pages/index.tsx

```diff
@@ -60,52 +60,69 @@ const Breadcrumbs: FunctionComponent = (): JSX.Element => (
 
 const AboutTheService: FunctionComponent = (): JSX.Element => (
   <>
-    <GovUKBody>Use this service to:</GovUKBody>
+    <GovUKBody>
+      You can use this service to register any of the following types of UK 406
+      Megahertz (MHz) emergency beacons:
+    </GovUKBody>
 
     <GovUKBulletedList>
+      <li>a Personal Locator Beacon (PLB) for any use</li>
       <li>
-        Register a single new 406 Megahertz (MHz) Personal Locator Beacon (PLB)
-        for use on maritime vessels
+        an Emergency Position Indicating Radio Beacon (EPIRB) for maritime use
       </li>
+      <li>an Emergency Locator Transmitter (ELT) for aircraft use</li>
+      <li>a Beacon dongle for aircraft use</li>
     </GovUKBulletedList>
 
-    <GovUKBody>Registering takes around 10 minutes.</GovUKBody>
+    <GovUKBody>Registering takes around 15 minutes.</GovUKBody>
 
-    <GovUKBody>
-      This service is only for UK programmed 406MHz beacons. You can{" "}
+    <InsetText>
+      If you want to register multiple beacons (i.e. more than 5),&nbsp;
       <AnchorLink href="mailto:ukbeacons@mcga.gov.uk">
         contact the UK Beacon Registry
       </AnchorLink>{" "}
-      if you aren{"'"}t sure if your beacon is 406MHz or not.
-    </GovUKBody>
-
-    <WarningText>
-      <>
-        You cannot register a beacon for non-maritime use through this service.
-        You can use another service to{" "}
-        <AnchorLink href="https://forms.dft.gov.uk/mca-sar-epirb/">
-          register a beacon for aircraft or land-based use.
-        </AnchorLink>
-      </>
-    </WarningText>
+      who will provide you with a spreadsheet template
+    </InsetText>
 
     <h2 className="govuk-heading-m">Before you start</h2>
```

Comment:
Now that we have a fancy new ✨ `SectionHeading` ✨ component 😂 we could use that here (and below for `Other ways to apply` and `Data protection regulations`?